packages/rum-core/src/common/utils.js

```diff
@@ -401,6 +401,36 @@ function isPerfTypeSupported(type) {
   )
 }
 
+/**
+ * The goal of this is to make sure that HAR files
+ * can be created containing beacons with the payload readable
+ */
+function isBeaconInspectionEnabled() {
+  const flagName = '_elastic_inspect_beacon_'
+
+  if (sessionStorage.getItem(flagName) !== null) {
```

Comment:
Why not look for values other than null or undefined?, I would change this to != null to cover for those two cases.

Reply:
Thanks for asking this. That is done in that why because storage (session and local) returns [null](https://developer.mozilla.org/en-US/docs/Web/API/Storage/getItem#return_value) when the key doesn't exist:

> A string containing the value of the key. If the key does not exist, null is returned.

Apart from that, the value of an item is always stored as a string, for instance:

```
sessionStorage.setItem('key', null) ---> stored as "null"
sessionStorage.setItem('key', undefined) ---> stored as "undefined"
// etc
```